config/routes.rb

```diff
@@ -3,7 +3,7 @@
   get 'trains/index'
   devise_for :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-  root to: "home#rootpage" 
+  root to: "home#root_page" 
   # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
```

Comment:
be care full about naming convention whenever you define files name and methods name

Reply:
okay i will make sure. thanks for review.